veggieseasons/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -460,8 +464,9 @@
 					"$(inherited)",
 					"$(PROJECT_DIR)/Flutter",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = io.flutter.veggieseasons;
+				PRODUCT_BUNDLE_IDENTIFIER = dev.flutter.VeggieSeasons;
```

Comment:
just to double check, this implies we need to remove the app on app store connect and recreate a new one (which is ok)

Reply:
Already done it, actually. I knew the website was moving from flutter.io to flutter.dev, so I figured this should reflect it.